src/subapps/studioLegacy/containers/StudioContainer.tsx

```diff
@@ -114,7 +116,7 @@ const StudioContainer: React.FunctionComponent = () => {
         studioId,
         studioResource._rev,
         {
-          ...studioResource,
+          ...omitBy(studioResource, (_, key) => key.trim().startsWith('_')),
```

Comment:
General question: Given the advancements in modern JavaScript, I'm wondering if we could lean more on native methods like filter and reduce for this case, rather than relying on lodash?